src/renderers/dom/server/ReactServerUpdateQueue.js

```diff
@@ -39,7 +39,7 @@ function warnNoop(publicInstance: ReactComponent<any, any, any>, callerName: str
  * @param {Transaction} transaction
  */
 class ReactServerUpdateQueue {
-  /* :: transaction: Transaction; */
+  transaction: Transaction;
 
   constructor(transaction: Transaction) {
```

Comment:
The flow team wants to make this an error in the future. We shouldn't be able to use a non-flow file as type as it silently turns it into any. (Internal discussion: https://www.facebook.com/groups/flowtype/permalink/1132421916806422/ )